document_page_approval/models/document_page.py

```diff
@@ -0,0 +1,146 @@
+# Copyright (C) 2013 Savoir-faire Linux (<http://www.savoirfairelinux.com>).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+
+from ast import literal_eval
+
+from odoo import api, fields, models
+
+
+class DocumentPage(models.Model):
+    """Useful to know the state of a document."""
+
+    _inherit = "document.page"
+
+    history_ids = fields.One2many(
+        order="approved_date DESC", domain=[("state", "=", "approved")]
+    )
```

Comment:
_order="approved_date DESC"

history_ids = fields.One2many(
         domain=[("state", "=", "approved")]
    )

Reply:
`order` or `_order` attribute does not work on a field.
I have added this ordering to the tree view with `default_order` attribute.

Reply:
I think there was a mistake. To correct: remove the field order attribute and add the _order attribute at the table level.

Reply:
But that's not the intention. The goal is to order the children model, so the other technique is the required one.